frontend/src/scenes/max/QuestionInput.tsx

```diff
@@ -90,6 +100,52 @@ export function QuestionInput({ isFloating }: QuestionInputProps): JSX.Element {
                             icon={threadLoading ? <IconStopFilled /> : <IconArrowRight />}
                         />
                     </div>
+                    {activeSuggestionGroup && !isFloating && (
+                        <ToggleGroup
+                            type="single"
+                            className="absolute inset-x-2 top-full grid auto-rows-auto p-1 border-x border-b rounded-b backdrop-blur-sm bg-[var(--glass-bg-3000)] z-10"
```

Comment:
For visual harmony, we need to round with `-lg`, because we're wrapping a collection of elements that are each rounded with the base radius – this looks _slightly_ off:

<img width="102" alt="Screenshot 2025-05-26 at 23 54 24" src="https://github.com/user-attachments/assets/17be1042-2c7a-410c-8e8f-0c36b37221b5" />

```suggestion
                            className="absolute inset-x-2 top-full grid auto-rows-auto p-1 border-x border-b rounded-lg-b backdrop-blur-sm bg-[var(--glass-bg-3000)] z-10"
```